<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>421</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(54, 54, 54);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>661</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="userListWidget">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>181</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(79, 79, 79);
border:2px solid rgb(255, 156, 16);
color: rgb(255, 156, 16);
border-radius: 5px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="SendLineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(79, 79, 79);
border:3px solid rgb(255, 156, 16);
color: rgb(255, 156, 16);
border-radius: 5px;
</string>
     </property>
     <property name="maxLength">
      <number>500</number>
     </property>
     <property name="placeholderText">
      <string>Введите сообщение...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Messages">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(79, 79, 79);
border:2px solid rgb(255, 156, 16);
color: rgb(255, 156, 16);
border-radius: 5px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>290</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:4px solid black;
border-radius:25px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QPushButton" name="SendButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(79, 79, 79);
border:3px solid rgb(255, 156, 16);
color: rgb(255, 156, 16);
border-radius: 25px;
</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="MyLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>5</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 170, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
